index.html

```diff
@@ -8,19 +8,16 @@
 	<body>
 		<header>
 			<div class="login">
-				<h1 class="logo">
-		        </h1>
+				<h1 class="logo"></h1>
 				<form>
 					<p>
-						<label id="label_username" for="username">Username or Email Address</label>
-		            	<input id="username" type="text" />
+						<label id="label_username" for="username">Username or Email Address</label><input id="username" type="text" />
 					</p>
 					<p>
-						<label id="label_pass" for="password">Password</label>
-		            	<input id="password" type="text" />
+						<label id="label_pass" for="password">Password</label><input id="password" type="text" />
+					</p>
 					<p class="forgetmenot">
-						<label id="label_remember" for="checkbox"> Remember Me</label>
-						<input id="checkbox" type="checkbox" />
+						<label id="label_remember" for="checkbox"> Remember Me</label><input id="checkbox" type="checkbox" />
```

Comment:
I think I prefer this on two lines too – and instead of a space before the "Remember Me" string, maybe you could use padding on the label instead?